src/User.cpp

```diff
@@ -45,8 +51,40 @@ void User::setFirecode(std::string firecode)
 {
     firecode_ = firecode;
 }
+void User::setPassword(std::string password)
+{
+    password_=password;
+}
+void User::setMail(std::string mail)
+{
+    mail_=mail;
+}
 
-std::string User::getNick() const 
+std::string User::getName() const
+{
+    return name_;
+}
+std::string User::getNick()  const
 {
     return nick_;
-}
\ No newline at end of file
+}
+std::string User::getPassword()  const
+{
+    return password_;
+}
+std::string User::getMail()  const
+{
+    return mail_;
+}
+std::string User::getGroup()  const
+{
+    return group_;
+}
+std::string User::getGitHub()  const
+{
+    return gitHub_;
+}
+std::string User::getFirecode()  const
+{
+    return firecode_;
+}
```

Comment:
If you would have a mentioned `serialize()` method, all those getters wouldn't be neccessary.